lib/katello/tasks/reset.rake

```diff
@@ -26,6 +26,14 @@ namespace :katello do
     end
   end
 
+  task :reset_default_smart_proxy do
+    hostname = Socket.gethostname.chomp
+    system("rake -- config -k ssl_ca_file -v /etc/pki/katello/certs/katello-default-ca.crt")
+    system("rake -- config -k ssl_certificate -v /etc/pki/katello/certs/#{hostname}-foreman-client.crt")
+    system("rake -- config -k ssl_priv_key -v /etc/pki/katello/private/#{hostname}-foreman-client.key")
+    SmartProxy.where(name: hostname, url: "https://#{hostname}:9090").first_or_create
```

Comment:
This isn't working for me for some reason, You might use .first_or_create!   to see if you can get it to tell you an error (for some reason i couldn't)